.Lib9c.Tests/Model/State/PetStateTest.cs

```diff
@@ -9,37 +9,56 @@
 
     public class PetStateTest
     {
-        [Fact]
-        public void Serialize()
+        [Theory]
+        [InlineData(int.MinValue)]
+        [InlineData(int.MaxValue)]
+        public void Serialize(int petId)
         {
-            var state = new PetState(1);
+            var state = new PetState(petId);
+            Assert.Equal(petId, state.PetId);
+            Assert.Equal(0, state.Level);
             var serialized = state.Serialize();
             var deserialized = new PetState((List)serialized);
-
             Assert.Equal(state.PetId, deserialized.PetId);
             Assert.Equal(state.Level, deserialized.Level);
+
+            var serialized2 = deserialized.Serialize();
+            Assert.Equal(serialized, serialized2);
         }
 
-        [Fact]
-        public void LevelUp()
+        [Theory]
+        [InlineData(int.MinValue, false)]
+        [InlineData(0, false)]
+        [InlineData(int.MaxValue, true)]
```

Comment:
I think we should test within pet's level range, not int min/max.

Reply:
There is only one case where `PetState.Level` to be negative. That is the constructor with serialized argument. And that case does not happen unless someone intend to.
But spec is spec.😅 So I wrote unit test like this.